boards/tenstorrent/tt_blackhole/doc/index.rst

```diff
@@ -91,28 +91,34 @@ Supported Features
    Some components are in the process of being migrated to Zephyr's driver model, and are
    currently implemented in the ``lib/tenstorrent/bh_arc`` library.
 
-.. :external+zephyr:zephyr:board-supported-hw::
```

Comment:
Does this necessarily need to be removed? IIRC, it did not cause docs to fail to build, but it wasn't rendering correctly for some reason.